### PR TITLE
feat: empty state in master data and publication service pages

### PR DESCRIPTION
**Pull request by doohanas**
### Overview
- Add empty state in master data and publication service pages

### Changes
- [chore: add two assets for empty state](https://github.com/jabardigitalservice/portal-jabar-cms/commit/c1e034b5d561e5b390aeb3c8fc8a4210592607a4)
- [feat (empty-state): to show empty state](https://github.com/jabardigitalservice/portal-jabar-cms/commit/e1a843c8451845ca26b1b6110780c15a4a1e4f98)
- [chore (constants): add data not found state for using in empty state](https://github.com/jabardigitalservice/portal-jabar-cms/commit/376e5ecf08e69d86e8e6a67a25f008f3b7ae6643)
- [feat (master-data-service): add empty state when the data isn't available](https://github.com/jabardigitalservice/portal-jabar-cms/commit/ecc5945472d351d0625e12eab7d1e5b910b010e0) 
- [feat (publication-service): add empty state component to show user when the publication data isn't available](https://github.com/jabardigitalservice/portal-jabar-cms/commit/10c80bff243c5ccc714956f6ba83719fdda66af8) 
- [refactor (master-data-service): remove justify-between](https://github.com/jabardigitalservice/portal-jabar-cms/commit/9adb5d76a24967095e2a5c2c6705280aefe9459c)

### Preview

1. Empty state `master-data-service` page
 
https://github.com/jabardigitalservice/portal-jabar-cms/assets/79241754/d5057560-8d5d-43b3-888c-8082bed99cdf

https://github.com/jabardigitalservice/portal-jabar-cms/assets/79241754/22cdd21c-a5b1-48c0-a30f-4620b1e1b1c1

2. Empty state `publication-service` page

https://github.com/jabardigitalservice/portal-jabar-cms/assets/79241754/b15fffd0-bbc6-47f8-8848-2c7b82608a86

https://github.com/jabardigitalservice/portal-jabar-cms/assets/79241754/3648123c-324f-44c0-94c7-0374de1fe83a

### Evidence
title: feat: empty state in master data and publication service pages
project: Portal Jabar
participants: @doohanas @Ibwedagama @yoslie

### CODE REVIEW

File: src/components/Services/Publication/index.vue [hunk 1]
```diff
@@ -167,8 +196,21 @@ export default {
   data() {
     return {
       currentTab: 'ALL',
+      dataNotFoundState: DATA_NOT_FOUND_STATE,
       formatDate,
       loading: false,
+      // isSearch use to check search feature is used or not
+      isSearch: false,
+      // isSearch use to check filter feature is used or not
```
Comment (Ibwedagama) [inline src/components/Services/Publication/index.vue:204]: this comment should be `isFilter use to check...` right? @doohanas

**Review by Ibwedagama — APPROVED** [review]

File: src/assets/images/data-not-found.svg
Comment (yoslie) [inline src/assets/images/data-not-found.svg:1]: hey @doohanas, 
I already added the same asset, kindly check `src/assets/icons/search-not-found.svg`

**[ related PR](https://github.com/jabardigitalservice/portal-jabar-cms/pull/454/files)

File: src/components/Services/Publication/index.vue (hunk 1 above)
Comment (doohanas) [reply src/components/Services/Publication/index.vue:204]: yes, i will change it

File: src/assets/images/data-not-found.svg
Comment (doohanas) [reply src/assets/images/data-not-found.svg:1]: hello @yoslie, oh I forget if you already added it. So, I will remove my assets 👍🏽

**Review by yoslie — APPROVED** [review]

**Pull request merged**